src/app/components/users/users.component.ts

```diff
@@ -31,7 +31,6 @@ export class UsersComponent implements OnInit {
     console.log(product);
     const dialogRef = this.dialog.open(EditUserDialogComponent, {
       width: '400px',
-      height: '540px',
```

Comment:
Hvorfor er dette blevet ændret? Dette PR er for at lave en reset password feature, ikke noget med users, så vidt jeg ved?

Reply:
Jeg kan godt se at det ikke giver mening at ændre i users når det er en login-branch. Grunden til at jeg har gjort det er at at efter at have slettet .mat-dialog-content i styles.scss, får denne dialog en fixed højde, som giver noget grimt whitespace. Ved at slette den faste højde her, wrappes dialogen rundt om indholdet.
Jeg synes det er lidt overkille at lave et PR for at slette en statisk højde.